src/main/java/dev/tenfont/cpmm/lang/components/TokenType.java

```diff
@@ -8,22 +8,108 @@
 @Getter
 public enum TokenType {
     // WHITESPACE
-    NEW_LINE(false, '\n')
+    NEW_LINE(false, '\n'),
+    WHITESPACE(false, ' '),
 
     // KEYWORDS
 
+    IF(true, "if"),
+    ELSE(true, "else"),
+    RETURN(true, "return"),
+    FUTURE(true, "future"),
+    PAST(true, "past"),
+
     // OPERATORS
 
+    ARITHMETIC_OPERATOR(true, reader -> {
+        switch (reader.peekChar()) {
+            case '+', '-', '/', '%' -> {
+                return String.valueOf(reader.readChar());
+            }
+            case '*' -> {
+                reader.readChar();
+                if (reader.canRead(1) && reader.peekChar() == '*') {
+                    reader.readChar();
+                    return "**";
+                }
+                return "*";
+            }
+        }
+        return null;
+    }),
+    COMPARISON_OPERATOR(true, reader -> switch (reader.peekChar()) {
+        case '>', '<' -> reader.readChar();
+        default -> null;
+    }),
```

Comment:
```suggestion
    COMPARISON_OPERATOR(true, reader -> {
        char next = reader.readChar();
        return (next == '<' || next == '>') ? next : null;
    }),
```
We can optimize this by reducing the amount of substrings to 1